skills/mindroom-docs/references/page__deployment__google-services-user-oauth__index.md

```
@@ -35,27 +35,25 @@ http://localhost:8765/api/oauth/google_gmail/callback
 
 ## Configure MindRoom
 
-For a single personal OAuth client, set the shared fallback client variables:
+For a single personal OAuth client, store shared Google OAuth app client config under `google_oauth_client` through the dashboard credentials API or raw credentials editor:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)